### ISSUE TITLE
EIP-1559 v2: small UI tweaks

### ISSUE THREAD

**Issue by kevinghim** [MetaMask/metamask-extension#12771]
Small UI tweaks based on @adonesky1 's feedback:

- [x] Remove GWEI amount in Advanced gas fee edit. Just show fiat. Add tooltip that shows full equation. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=460:5848#125751387)
- [ ] Add `x` multiplier symbol after multiplier amount. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=460:5848#126029907) 
- [ ] Remove multiplier symbol after GWEI amount in `Edit in GWEI` mode. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=915:11206#125745003)
- [x] Vertically align gas option emojis with gas option text. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126059483)
- [x] Add hover state to `Edit gas fee` row. Color is #EAF6FF [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=460%3A5632)
- [x] Add select state to `Edit gas fee` row. Color is #F2F3F4 [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=460%3A5632)
- [x] Left align values to the column headers in `Edit gas fee` screen [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126056825) 
- [x] Increase emoji size to 20px in `Edit gas fee` screen. Text labels can remain 12px. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126056639) 
- [x] Make the `Gas option` column slightly smaller and increase the space between the `Time` and `Max fee` columns [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126055694)
- [x] Add hover state to each gas option select. Color should be slightly darker PRIMARY3 blue color. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126054900)
- [x]  If there is no “Site” recommended price within the gas options selection, we shouldn’t show that line at all (not grayed out, just not there).
- [x] The “Advanced >” and “Edit” are equally spaced and look like one group. For all options, let’s reduce the space between the label and “>” and also increase the space between “Advanced >” and “Edit”. Mock up below.
<img width="583" alt="Screen Shot 2021-11-30 at 9 36 07 AM" src="https://user-images.githubusercontent.com/6951929/144135239-84a8e3d2-2c5a-48e5-ac8c-4c2aff7333dd.png">

**Comment by jpuri** [comment]
@kevinghim , @jakehaugen : for point 1 above can you plz share some details about how the tooltip will look.

**Comment by jpuri** [comment]
Screens after implementing design fixes:
<img width="394" alt="Screenshot 2021-12-02 at 9 43 04 AM" src="https://user-images.githubusercontent.com/2182307/144356403-a55aea51-39d4-4f45-a5ff-e443e5aeb993.png">
<img width="393" alt="Screenshot 2021-12-02 at 9 37 50 AM" src="https://user-images.githubusercontent.com/2182307/144356410-f1747488-3594-4675-a5a9-184d75f8fbc8.png">

**Comment by jakehaugen** [comment]
> * point 1 above can you plz share some details about how the tooltip will look.

This is referring to the "Max base fee" input. Tooltip style should be the same (next to the label). Copy update to: `When your transaction gets included in the block, any difference between your max base fee and the actual base fee will be refunded. Total amount is calculated as max base fee (in GWEI) * gas limit.`

> * point-2 is not achievable with out input component

OK, I think we can wait on making this update. 

> * select hover state with primary3 colour in background looks like, should I change test colour also ?

Let's try PRIMARY 2 as the color and no text color change. I mocked this up [here](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=1387%3A14587). 

Not that the hover state should apply to all items -- selected or not.

**Comment by jpuri** [comment]
Hey @jakehaugen : thanks for the feedback.

I have implemented hover state for all row which is primary-2:
<img width="471" alt="Screenshot 2021-12-03 at 11 55 19 AM" src="https://user-images.githubusercontent.com/2182307/144556131-9841a35c-ec62-4d92-a5f2-014947e38a69.png">

This is how tooltip looks now:
<img width="381" alt="Screenshot 2021-12-03 at 12 11 31 PM" src="https://user-images.githubusercontent.com/2182307/144557454-b244c5ed-6715-4f1d-a764-dc16d75db7cb.png">

**Comment by jpuri** [comment]
Hi @kevinghim : all points in this issue are addressed and I have moved it for review. For any new issue can you plz create a separate card now.
